blocks/core/ff_module/ff_module-columnar-list/ff_module-columnar-list.less

```diff
@@ -24,6 +24,9 @@
 		width: 100%;
 		padding: @ff_size_control_vertical/2 0;
 	}
+	.ie10 &__item {
```

Comment:
We don't have browser targeting prefixes in the patternlib yet, and have been trying to avoid having to add them - is there any other way this could be solved without targeting?